src/ProgressMeter/LICENSE.txt

```diff
@@ -6,3 +6,15 @@ Permission is hereby granted, free of charge, to any person obtaining a copy of
 The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+> For the function `isnothing`
+
+Copyright (c) 2009-2019: Jeff Bezanson, Stefan Karpinski, Viral B. Shah, and other contributors:
```

Comment:
Actually, I missed this part. I don't think we need this for 2-line code.

FSF says

> A change of just a few lines (less than 15 or so) is not legally significant for copyright. 

https://www.gnu.org/prep/maintain/html_node/Legally-Significant.html

I found the link via:
https://discourse.julialang.org/t/copyright-issues-for-code-excerpts/17868/3
https://discourse.julialang.org/t/i-asked-for-help-optimizing-my-julia-code-now-its-someones-package/34679/11

Reply:
thanks, good to know! reverted the license change